_events/sr2019/competition.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Competition
+date: 2019-04-06 9:00:00
+layout: event
+type: competition
+location: Southampton University Highfield Campus
+---
+
+The Student Robotics Competition marks the ultimate stage of SR2019, showcasing the months of preparation. Our teams will work hard to push their robots to the limit and quickly make repairs between league matches. We will finish the weekend with the finals where the best bots will be named victorious.
+
+## Directions
+
+The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. 
+
+* How to [get to campus][soton-campus-directions]
+
+## Schedule
+
+The competition will be held from *Saturday 6th April* until *Sunday 7th April*. The exact start/end times are to be confirmed, but doors open 9am, and we aim to finish around 6pm.
+
+## More details
+
+To come in the weeks following the date announcement.
```

Comment:
This reads oddly to me, I think because the the announcement it mentions is in the past (or will be by the time this is read). Perhaps just say something like "coming soon"?